test/provider/immoscout-mobile.test.js

```diff
@@ -0,0 +1,34 @@
+import {expect} from 'chai';
+import {convertWebToMobile} from '../../lib/provider/immoscout-mobile.js';
+
+describe('#immoscout-mobile testsuite()', () => {
```

Comment:
A good test case, however can you please make sure to properly test the provider (see the other provider tests, you can pretty much copy and paste the existing immoscout test)

Reply:
Added a provider test based on the other providers. I've decided to split the test suite into the provider test and URL conversion unit tests to avoid issues with `provider.init(...)` and similarity cache (stateful) dependencies.